server/views/manage/resident/residentProfile.njk

```diff
@@ -48,9 +48,11 @@
     {% endif %}
 
     <div class="govuk-grid-column-three-quarters">
-    {% if cardList %}
-        {% include "./partials/cardList.njk" %}
-    {% endif %}
+      {% if sectionTemplate %}
+        {% include sectionTemplate %}
+      {% elseif cardList %}
+        {% include "manage/resident/partials/cardList.njk" %}
```

Comment:
I'm wondering why the full path here?   It's not a problem, but I don't see why it's needed.